src/main/resources/kafka-topics.json

```diff
@@ -4,6 +4,11 @@
       "name": "inventory.instance",
       "numPartitions": 1,
       "replicationFactor": 1
+    },
+    {
+      "name": "inventory.item",
+      "numPartitions": 50,
```

Comment:
Why are items partitioned differently to instances?

Reply:
We will update the property for instances soon. This will be the same number.

Reply:
Ok